libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/security/Encryptor.java

```diff
@@ -58,67 +55,8 @@
     private static final String US_ASCII = "US-ASCII";
     private static final String PREFER_CIPHER_TRANSFORMATION = "AES/CBC/PKCS5Padding";
     private static final String MAC_TRANSFORMATION = "HmacSHA256";
-    private static String bestCipherAvailable;
-    private static boolean isFileSystemEncrypted;
-
-    /**
-     * Initializes this module.
-     *
-     * @param ctx Context.
-     * @return True - if the cryptographic module was successfully initialized, False - otherwise.
-     */
-    public static boolean init(Context ctx) {
```

Comment:
Most of this code was legacy code that's not required anymore for 2 reasons:
1. File system encryption is available by default even with our `minApi`, i.e. `API 21`.
2. The reason we had hardcoded `BouncyCastle` in here was that it wasn't the default or even available on some devices prior to `Eclair` or `Froyo` (`Android 2.x`), which we haven't supported in years now.